assets/vue/components/Identicon.vue

```diff
@@ -18,13 +18,16 @@ export default {
   name: "Identicon",
   props: {
     size: Number,
-    text: String,
     src: String,
   },
   computed: {
     alt() {
       return `A ${this.size}x${this.size} identicon from '${this.text}'`;
     },
+    text() {
+      const [_, encodedText] = this.src.split("/");
+      return decodeURI(encodedText);
```

Comment:
This doesn't _exactly_ correspond to the image being updated because there is a small additional wait for the http for the `src` to return. This does exactly correspond to the value for `this.src` being updated in the client. Cool?